### PR TITLE
PLU-309: [TILES-ATOMIC-INCREMENT-3] enhance multirow-multicol UI with custom styles and operators

### PR DESCRIPTION
**Pull request by pregnantboy**
## TLDR;
FRONTEND-ONLY
For update-row action, allow users to set increment or decrement existing value.

## What changed
- Introduce 2 new attributes to dropdown fields, `isSearchable` and `subFields.customStyles`
- if `isSearchable` is set to false, it prevent users from typing to search and center aligns the content (this is an existing prop that is unused)
- `subFields.customStyles` allows users to override the container of the subfields in `multirow-multicol` fields 

## Why
This frontend change will allow users to change the way the value is modified, either set-as, add by or subtract by.

### CODE REVIEW

**Comment by pregnantboy** [comment]
* **#835** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/835?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#819** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/819?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/819?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#818** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/818?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#817** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/817?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop-v2`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by pregnantboy — COMMENTED** [review]
FYI to reviewers, this PR does not include backend changes (i.e. setting the operator to any value will still result in a "set as" operation)

File: packages/backend/src/apps/custom-api/actions/http-request/index.ts [hunk 1]
```diff
@@ -61,13 +61,15 @@ const action: IRawAction = {
...
           variables: false,
+          customStyle: { flex: 2 },
         },
         {
           placeholder: 'Value',
           key: 'value',
           type: 'string' as const,
           required: true,
           variables: true,
+          customStyle: { flex: 3 },
```
Comment (m0nggh) [inline packages/backend/src/apps/custom-api/actions/http-request/index.ts]: i think need to add `minWidth: 0` or else this happens
image

File: packages/backend/src/apps/tiles/actions/update-row/index.ts [hunk 2]
```diff
@@ -79,13 +79,34 @@ const action: IRawAction = {
...
+          ],
+          customStyle: { flexBasis: '71px' },
         },
         {
           placeholder: 'Value',
           key: 'cellValue',
           type: 'string' as const,
           required: false,
           variables: true,
+          customStyle: { flex: 3 },
```
Comment (m0nggh) [inline packages/backend/src/apps/tiles/actions/update-row/index.ts]: same thing for this

File: packages/backend/src/graphql/schema.graphql [hunk 3]
```diff
@@ -183,8 +183,10 @@ type ActionSubstepArgument {
   variables: Boolean
   variableTypes: [String]
   allowArbitrary: Boolean
+  isSearchable: Boolean
```
Comment (m0nggh) [inline packages/backend/src/graphql/schema.graphql:186]: can i check is this just for the pointer icon to be displayed if it is false? for sanity purposes, should we also include this in the `TriggerSubstepArgument` in case we forget

**Review by m0nggh — COMMENTED** [review]
other than the small changes, think its good

File: packages/backend/src/graphql/schema.graphql (hunk 3 above)
Comment (pregnantboy) [reply packages/backend/src/graphql/schema.graphql:186]: I feel like we just add in if necessary

**Review by kevinkim-ogp — APPROVED** [review]
lgtm 💪 
changes suggested by @m0nggh have been implemented and tested to work

**Comment by pregnantboy** [comment]
### Merge activity

* **Mar 10, 2:47 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/819).
* **Mar 10, 2:48 AM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/819).
